packaging/gaphor.spec

```diff
@@ -33,13 +33,13 @@ exe = EXE(pyz,
           a.scripts,
           options=[],
           exclude_binaries=True,
-          name='gaphor',
+          name='gaphor-exe',
```

Comment:
Why the name change?

Reply:
In Windows, the executable gets a .exe extension. But for Linux and macOS the executable doesn't get an extension. This causes the executable name to conflict with the name of the gaphor directory. We could call it something else if you have any ideas.

Reply:
Ah, I see. That sounds fair.